addons/mass_mailing/models/mailing.py

```diff
@@ -254,14 +254,6 @@ def write(self, values):
             values['body_html'] = self._convert_inline_images_to_urls(values['body_html'])
         return super(MassMailing, self).write(values)
 
-    @api.returns('self', lambda value: value.id)
-    def copy(self, default=None):
-        self.ensure_one()
-        default = dict(default or {},
-                       name=_('%s (copy)') % self.name,
-                       contact_list_ids=self.contact_list_ids.ids)
-        return super(MassMailing, self).copy(default=default)
-
```

Comment:
I guess this is a fix because of your first commit. Then, fixup it into the first commit.

Reply:
commits squashed